self-api/src/main/java/com/selfxdsd/api/Organization.java

```diff
@@ -0,0 +1,64 @@
+package com.selfxdsd.api;
+
+import javax.json.JsonObject;
+
+/**
+ * Provider organization of which authenticated User has admin rights.
+ * @author criske
+ * @version $Id$
+ * @since 0.0.8
+ */
+public interface Organization {
+
+    /**
+     * Organization ID.
+     * @return String.
+     */
+    String organizationId();
+
+    /**
+     * Organization repos.
+     * @return OrganizationRepos.
+     */
+    OrganizationRepos repos();
+
+    /**
+     * The Organization in JSON format as returned by the provider's API.
+     * @return JsonObject.
+     */
+    JsonObject json();
+
+    /**
+     * Extra information about the organization like name,
+     * description etc...
+     * @return Detail
+     */
+    Detail detail();
+
+    /**
+     * Contains extra information about the organization like name,
+     * description etc...
+     */
+    interface Detail {
```

Comment:
@criske Detail is not needed for now. The json() method is enough if we want some specific info about the Organization... we will be interested only in the repos, most likely.